Analytics Dashboard not accessible in Docker Desktop

**Description:**
Unable to access the analytics dashboard. The redirection to login fails with the following error. 
```
WARN {org.wso2.msf4j.internal.MSF4JHttpConnectorListener} - Unmapped exception feign.RetryableException: Connection refused (Connection refused) executing GET https://127.0.0.1:32293/api/am/admin/v1/custom-urls/carbon.super                                            
    at feign.FeignException.errorExecuting(FeignException.java:67)                                                                                                                                                                                                                                    
    at feign.SynchronousMethodHandler.executeAndDecode(SynchronousMethodHandler.java:104)                                                                                                                                                                                                             
    at feign.SynchronousMethodHandler.invoke(SynchronousMethodHandler.java:76)                                                                                                                                                                                                                        
    at feign.ReflectiveFeign$FeignInvocationHandler.invoke(ReflectiveFeign.java:103)                                                                                                                                                                                                                  
    at com.sun.proxy.$Proxy77.getCustomUrlInfo(Unknown Source)                                                                                                                                                                                                                                        
    at org.wso2.analytics.apim.idp.client.ApimIdPClient.getCustomUrlInfo(ApimIdPClient.java:721)                                                                                                                                                                                                      
    at org.wso2.analytics.apim.idp.client.ApimIdPClient.login(ApimIdPClient.java:294)                                                                                                                                                                                                                 
    at org.wso2.carbon.analytics.auth.rest.api.impl.LoginApiServiceImpl.loginAppNamePost(LoginApiServiceImpl.java:147)                                                                                                                                                                                
    at org.wso2.carbon.analytics.auth.rest.api.LoginApi.loginAppNamePost(LoginApi.java:82)                                                                                                                                                                                                            
    at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)                                                                                                                                                                                                                 
    at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)                                                                                                                                                                                               
    at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)                                                                                                                                                                                       
    at java.base/java.lang.reflect.Method.invoke(Method.java:566)                                                                                                                                                                                                                                     
    at org.wso2.msf4j.internal.router.HttpMethodInfo.invokeResource(HttpMethodInfo.java:187)                                                                                                                                                                                                          
    at org.wso2.msf4j.internal.router.HttpMethodInfo.invoke(HttpMethodInfo.java:143)                                                                                                                                                                                                                 
    at org.wso2.msf4j.internal.MSF4JHttpConnectorListener.dispatchMethod(MSF4JHttpConnectorListener.java:218)                                                                                                                                                                                       
    at org.wso2.msf4j.internal.MSF4JHttpConnectorListener.lambda$onMessage$47(MSF4JHttpConnectorListener.java:129)                                                                                                                                                                                    
    at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)                                                                                                                                                                                                      
    at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)                                                                                                                                                                                                      
    at java.base/java.lang.Thread.run(Thread.java:834)                                                                                                                                                                                                                                                
Caused by: java.net.ConnectException: Connection refused (Connection refused)                                                                                                                                                                                                                         
    at java.base/java.net.PlainSocketImpl.socketConnect(Native Method)                                                                                                                                                                                                                                
    at java.base/java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:399)                                                                                                                                                                                                        
    at java.base/java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:242)                                                                                                                                                                                                  
    at java.base/java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:224)                                                                                                                                                                                                           
    at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:403)                                                                                                                                                                                                                           
    at java.base/java.net.Socket.connect(Socket.java:609)                                                                                                                                                                                                                                             
    at java.base/sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:289)                                                                                                                                                                                                                     
    at java.base/sun.net.NetworkClient.doConnect(NetworkClient.java:177)                                                                                                                                                                                                                              
    at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:474)                                                                                                                                                                                                                          
    at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:569)                                                                                                                                                                                                                          
    at java.base/sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:265)                                                                                                                                                                                                                  
    at java.base/sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:372)                                                                                                                                                                                                                     
    at java.base/sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:191)                                                                                                                                                          
    at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1187)                                                                                                                                                                                               
    at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1081)                                                                                                                                                                                                
    at java.base/sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:177)                                                                                                                                                                   
    at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1592)                                                                                                                                                                                             
    at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1520)                                                                                                                                                                                              
    at java.base/java.net.HttpURLConnection.getResponseCode(HttpURLConnection.java:527)                                                                                                                                                                                                               
    at java.base/sun.net.www.protocol.https.HttpsURLConnectionImpl.getResponseCode(HttpsURLConnectionImpl.java:334)                                                                                                                                                                                   
    at feign.Client$Default.convertResponse(Client.java:152)                                                                                                                                                                                                                                          
    at feign.Client$Default.execute(Client.java:74)                                                                                                                                                                                                                                                   
    at feign.SynchronousMethodHandler.executeAndDecode(SynchronousMethodHandler.java:97)                                                                                                                                                                                                              
    ... 18 more
```

**Affected Product Version:**
3.2.0

**OS, DB, other environment details and versions:**  
Docker for Desktop

--- a/simple/README.md
+++ b/simple/README.md
@@ -39,9 +39,9 @@ chmod +x SIMPLIFIED_SCRIPT
 ```
 
 5. Open a browser and go to 
-- https://<NODE_IP>:32293/carbon/
-- https://<NODE_IP>:32293/publisher/
-- https://<NODE_IP>:32293/devportal/
+- https://<NODE_IP>:30443/carbon/
+- https://<NODE_IP>:30443/publisher/
+- https://<NODE_IP>:30443/devportal/
 - https://<NODE_IP>:30646/analytics-dashboard/
 
 use username: admin and password: admin as default login credentials.

--- a/simple/deployment-scripts/wso2am-ga.sh
+++ b/simple/deployment-scripts/wso2am-ga.sh
@@ -3096,11 +3096,11 @@ spec:
                 command:  ['sh', '-c', '${WSO2_SERVER_HOME}/bin/dashboard.sh stop']
           resources:
             requests:
-              memory: 4Gi
-              cpu: 2000m
+              memory: 1Gi
+              cpu: 1000m
             limits:
-              memory: 4Gi
-              cpu: 2000m
+              memory: 1Gi
+              cpu: 1000m
           imagePullPolicy: Always
           securityContext:
             runAsUser: 802
@@ -3768,11 +3768,11 @@ spec:
                 command:  ['sh', '-c', '${WSO2_SERVER_HOME}/bin/worker.sh stop']
           resources:
             requests:
-              memory: 4Gi
-              cpu: 2000m
+              memory: 1Gi
+              cpu: 1000m
             limits:
-              memory: 4Gi
-              cpu: 2000m
+              memory: 1Gi
+              cpu: 1000m
           imagePullPolicy: Always
           securityContext:
             runAsUser: 802
@@ -4200,10 +4200,10 @@ spec:
           resources:
             requests:
               memory: 2Gi
-              cpu: 2000m
+              cpu: 1000m
             limits:
-              memory: 3Gi
-              cpu: 3000m
+              memory: 2Gi
+              cpu: 1000m
           imagePullPolicy: Always
           ports:
             -

--- a/simple/deployment-scripts/wso2am-ga.sh
+++ b/simple/deployment-scripts/wso2am-ga.sh
@@ -2983,9 +2983,9 @@ data:
         allScopes: apim_analytics:admin openid apim:api_view apim:subscribe apim_analytics:monitoring_dashboard:own apim_analytics:monitoring_dashboard:edit apim_analytics:monitoring_dashboard:view apim_analytics:business_analytics:own apim_analytics:business_analytics:edit apim_analytics:business_analytics:view apim_analytics:api_analytics:own apim_analytics:api_analytics:edit apim_analytics:api_analytics:view apim_analytics:application_analytics:own apim_analytics:application_analytics:edit apim_analytics:application_analytics:view
         adminUsername: admin
         adminPassword: admin
-        kmDcrUrl: https://"ip.node.k8s.&.wso2.apim":32293/client-registration/v0.17/register
-        kmTokenUrlForRedirection: https://"ip.node.k8s.&.wso2.apim":32293/oauth2
-        kmTokenUrl: https://"ip.node.k8s.&.wso2.apim":32293/oauth2
+        kmDcrUrl: https://wso2am-pattern-1-am-1-service:9443/client-registration/v0.17/register
+        kmTokenUrlForRedirection: https://"ip.node.k8s.&.wso2.apim":30443/oauth2
+        kmTokenUrl: https://wso2am-pattern-1-am-1-service:9443/oauth2
         kmUsername: admin
         kmPassword: admin
         portalAppContext: analytics-dashboard
@@ -2993,9 +2993,9 @@ data:
         cacheTimeout: 900
         baseUrl: https://"ip.node.k8s.&.wso2.apim":30646
         grantType: authorization_code
-        publisherUrl: https://"ip.node.k8s.&.wso2.apim":32293
-        devPortalUrl: https://"ip.node.k8s.&.wso2.apim":32293
-        externalLogoutUrl: https://"ip.node.k8s.&.wso2.apim":32293/oidc/logout
+        publisherUrl: https://wso2am-pattern-1-am-1-service:9443
+        devPortalUrl: https://wso2am-pattern-1-am-1-service:9443
+        externalLogoutUrl: https://"ip.node.k8s.&.wso2.apim":30443/oidc/logout
 
     wso2.dashboard:
       roles:
@@ -3826,7 +3826,7 @@ data:
     [server]
     hostname = ""ip.node.k8s.&.wso2.apim""
     node_ip = "$env{NODE_IP}"
-    offset=22850
+    # offset=0
     mode = "single" #single or ha
     base_path = "${carbon.protocol}://${carbon.host}:${carbon.management.port}"
     #discard_empty_caches = false
@@ -3971,7 +3971,7 @@ data:
     #enable_token_hashing = false
 
     [apim.devportal]
-    url = "https://"ip.node.k8s.&.wso2.apim":32293/devportal"
+    url = "https://"ip.node.k8s.&.wso2.apim":30443/devportal"
     #enable_application_sharing = false
     #if application_sharing_type, application_sharing_impl both defined priority goes to application_sharing_impl
     #application_sharing_type = "default" #changed type, saml, default #todo: check the new config for rest api
@@ -4085,6 +4085,9 @@ data:
     username = "${admin.username}"
     password = "${admin.password}"
     'header.X-WSO2-KEY-MANAGER' = "default"
+
+    [transport.https.properties]
+    proxyPort = 30443
 ---
 
 apiVersion: v1
@@ -4103,31 +4106,27 @@ spec:
     -
       name: pass-through-http
       protocol: TCP
-      # for 8280 with a offset of 22850
-      port: 31130
-      targetPort: 31130
+      port: 8280
+      targetPort: 8280
       nodePort: 30280
     -
       name: pass-through-https
       protocol: TCP
-      # for 8243 with a offset of 22850
-      port: 31093
-      targetPort: 31093
+      port: 8243
+      targetPort: 8243
       nodePort: 30243
     -
       name: servlet-http
       protocol: TCP
-      # for 9763 with a offset of 22850
-      port: 32613
-      targetPort: 32613
+      port: 9763
+      targetPort: 9763
       nodePort: 30763
     -
       name: servlet-https
       protocol: TCP
-      # for 9443 with a offset of 22850
-      port: 32293
-      targetPort: 32293
-      nodePort: 32293
+      port: 9443
+      targetPort: 9443
+      nodePort: 30443
 ---
 
 apiVersion: apps/v1
@@ -4182,7 +4181,7 @@ spec:
               command:
                 - /bin/sh
                 - -c
-                - nc -z localhost 32293
+                - nc -z localhost 9443
             initialDelaySeconds: 250
             periodSeconds: 10
           readinessProbe:
@@ -4190,7 +4189,7 @@ spec:
               command:
                 - /bin/sh
                 - -c
-                - nc -z localhost 32293
+                - nc -z localhost 9443
             initialDelaySeconds: 250
             periodSeconds: 10
           lifecycle:
@@ -4199,24 +4198,24 @@ spec:
                 command:  ['sh', '-c', '${WSO2_SERVER_HOME}/bin/wso2server.sh stop']
           resources:
             requests:
-              memory: 2Gi
+              memory: 1Gi
               cpu: 1000m
             limits:
               memory: 2Gi
-              cpu: 1000m
+              cpu: 2000m
           imagePullPolicy: Always
           ports:
             -
-              containerPort: 31130
+              containerPort: 8280
               protocol: "TCP"
             -
-              containerPort: 31093
+              containerPort: 8243
               protocol: "TCP"
             -
-              containerPort: 32613
+              containerPort: 9763
               protocol: "TCP"
             -
-              containerPort: 32293
+              containerPort: 9443
               protocol: "TCP"
           env:
             - name: HOST_NAME
@@ -4330,6 +4329,9 @@ function get_node_ip(){
       if [[ $(kubectl config current-context) = "minikube" ]]
       then
           NODE_IP=$(minikube ip)
+      elif [[ $(kubectl config current-context) = "docker-desktop" ]]
+      then
+          NODE_IP="127.0.0.1"
       else
         echo "We could not find your cluster node-ip."
         while [[ -z "$NODE_IP" ]]
@@ -4353,6 +4355,9 @@ function progress_bar(){
   pod_status=$(kubectl get pods -n wso2 -o jsonpath='{.items[?(@.metadata.labels.product=="api-manager")].status.conditions[*].status}')
 
   num_true_const=0; progress_unit="";num_true=0; time_proc=0;
+
+  # check for 250s until the pod is live.
+  time_limit=250 
 
   arr_dep=($dep_status); arr_pod=($pod_status)
 
@@ -4482,10 +4487,10 @@ function deploy(){
     echoBold "Successfully deployed WSO2 API Manager.\n\n"
 
     echoBold "1. Try navigating to\n\n"
-    echoBold "\thttps://$NODE_IP:32293/carbon/\n"
-    echoBold "\thttps://$NODE_IP:32293/publisher/\n"
-    echoBold "\thttps://$NODE_IP:32293/devportal/\n"
-    echoBold "\thttps://$NODE_IP:30643/analytics-dashboard/\n\n"
+    echoBold "\thttps://$NODE_IP:30443/carbon/\n"
+    echoBold "\thttps://$NODE_IP:30443/publisher/\n"
+    echoBold "\thttps://$NODE_IP:30443/devportal/\n"
+    echoBold "\thttps://$NODE_IP:30646/analytics-dashboard/\n\n"
     echoBold "from your favourite browser using credentials admin/admin\n\n"
 
     echoBold "2. Follow \"https://apim.docs.wso2.com/en/latest/GettingStarted/quick-start-guide/\" to start using WSO2 API Manager.\n\n"

--- a/simple/deployment-scripts/wso2am-latest.sh
+++ b/simple/deployment-scripts/wso2am-latest.sh
@@ -3106,11 +3106,11 @@ spec:
                 command:  ['sh', '-c', '${WSO2_SERVER_HOME}/bin/dashboard.sh stop']
           resources:
             requests:
-              memory: 4Gi
-              cpu: 2000m
+              memory: 1Gi
+              cpu: 1000m
             limits:
-              memory: 4Gi
-              cpu: 2000m
+              memory: 1Gi
+              cpu: 1000m
           imagePullPolicy: Always
           securityContext:
             runAsUser: 802
@@ -3778,11 +3778,11 @@ spec:
                 command:  ['sh', '-c', '${WSO2_SERVER_HOME}/bin/worker.sh stop']
           resources:
             requests:
-              memory: 4Gi
-              cpu: 2000m
+              memory: 1Gi
+              cpu: 1000m
             limits:
-              memory: 4Gi
-              cpu: 2000m
+              memory: 1Gi
+              cpu: 1000m
           imagePullPolicy: Always
           securityContext:
             runAsUser: 802
@@ -4212,8 +4212,8 @@ spec:
               memory: 2Gi
               cpu: 2000m
             limits:
-              memory: 3Gi
-              cpu: 3000m
+              memory: 2Gi
+              cpu: 2000m
           imagePullPolicy: Always
           ports:
             -

--- a/simple/deployment-scripts/wso2am-latest.sh
+++ b/simple/deployment-scripts/wso2am-latest.sh
@@ -2993,9 +2993,9 @@ data:
         allScopes: apim_analytics:admin openid apim:api_view apim:subscribe apim_analytics:monitoring_dashboard:own apim_analytics:monitoring_dashboard:edit apim_analytics:monitoring_dashboard:view apim_analytics:business_analytics:own apim_analytics:business_analytics:edit apim_analytics:business_analytics:view apim_analytics:api_analytics:own apim_analytics:api_analytics:edit apim_analytics:api_analytics:view apim_analytics:application_analytics:own apim_analytics:application_analytics:edit apim_analytics:application_analytics:view
         adminUsername: admin
         adminPassword: admin
-        kmDcrUrl: https://"ip.node.k8s.&.wso2.apim":32293/client-registration/v0.17/register
-        kmTokenUrlForRedirection: https://"ip.node.k8s.&.wso2.apim":32293/oauth2
-        kmTokenUrl: https://"ip.node.k8s.&.wso2.apim":32293/oauth2
+        kmDcrUrl: https://wso2am-pattern-1-am-1-service:9443/client-registration/v0.17/register
+        kmTokenUrlForRedirection: https://"ip.node.k8s.&.wso2.apim":30443/oauth2
+        kmTokenUrl: https://wso2am-pattern-1-am-1-service:9443/oauth2
         kmUsername: admin
         kmPassword: admin
         portalAppContext: analytics-dashboard
@@ -3003,9 +3003,9 @@ data:
         cacheTimeout: 900
         baseUrl: https://"ip.node.k8s.&.wso2.apim":30646
         grantType: authorization_code
-        publisherUrl: https://"ip.node.k8s.&.wso2.apim":32293
-        devPortalUrl: https://"ip.node.k8s.&.wso2.apim":32293
-        externalLogoutUrl: https://"ip.node.k8s.&.wso2.apim":32293/oidc/logout
+        publisherUrl: https://wso2am-pattern-1-am-1-service:9443
+        devPortalUrl: https://wso2am-pattern-1-am-1-service:9443
+        externalLogoutUrl: https://"ip.node.k8s.&.wso2.apim":30443/oidc/logout
 
     wso2.dashboard:
       roles:
@@ -3836,7 +3836,7 @@ data:
     [server]
     hostname = ""ip.node.k8s.&.wso2.apim""
     node_ip = "$env{NODE_IP}"
-    offset=22850
+    # offset=0
     mode = "single" #single or ha
     base_path = "${carbon.protocol}://${carbon.host}:${carbon.management.port}"
     #discard_empty_caches = false
@@ -3981,7 +3981,7 @@ data:
     #enable_token_hashing = false
 
     [apim.devportal]
-    url = "https://"ip.node.k8s.&.wso2.apim":32293/devportal"
+    url = "https://"ip.node.k8s.&.wso2.apim":30443/devportal"
     #enable_application_sharing = false
     #if application_sharing_type, application_sharing_impl both defined priority goes to application_sharing_impl
     #application_sharing_type = "default" #changed type, saml, default #todo: check the new config for rest api
@@ -4095,6 +4095,9 @@ data:
     username = "${admin.username}"
     password = "${admin.password}"
     'header.X-WSO2-KEY-MANAGER' = "default"
+
+    [transport.https.properties]
+    proxyPort = 30443
 ---
 
 apiVersion: v1
@@ -4113,31 +4116,27 @@ spec:
     -
       name: pass-through-http
       protocol: TCP
-      # for 8280 with a offset of 22850
-      port: 31130
-      targetPort: 31130
+      port: 8280
+      targetPort: 8280
       nodePort: 30280
     -
       name: pass-through-https
       protocol: TCP
-      # for 8243 with a offset of 22850
-      port: 31093
-      targetPort: 31093
+      port: 8243
+      targetPort: 8243
       nodePort: 30243
     -
       name: servlet-http
       protocol: TCP
-      # for 9763 with a offset of 22850
-      port: 32613
-      targetPort: 32613
+      port: 9763
+      targetPort: 9763
       nodePort: 30763
     -
       name: servlet-https
       protocol: TCP
-      # for 9443 with a offset of 22850
-      port: 32293
-      targetPort: 32293
-      nodePort: 32293
+      port: 9443
+      targetPort: 9443
+      nodePort: 30443
 ---
 
 apiVersion: apps/v1
@@ -4192,7 +4191,7 @@ spec:
               command:
                 - /bin/sh
                 - -c
-                - nc -z localhost 32293
+                - nc -z localhost 9443
             initialDelaySeconds: 250
             periodSeconds: 10
           readinessProbe:
@@ -4200,7 +4199,7 @@ spec:
               command:
                 - /bin/sh
                 - -c
-                - nc -z localhost 32293
+                - nc -z localhost 9443
             initialDelaySeconds: 250
             periodSeconds: 10
           lifecycle:
@@ -4209,24 +4208,24 @@ spec:
                 command:  ['sh', '-c', '${WSO2_SERVER_HOME}/bin/wso2server.sh stop']
           resources:
             requests:
-              memory: 2Gi
-              cpu: 2000m
+              memory: 1Gi
+              cpu: 1000m
             limits:
               memory: 2Gi
               cpu: 2000m
           imagePullPolicy: Always
           ports:
             -
-              containerPort: 31130
+              containerPort: 8280
               protocol: "TCP"
             -
-              containerPort: 31093
+              containerPort: 8243
               protocol: "TCP"
             -
-              containerPort: 32613
+              containerPort: 9763
               protocol: "TCP"
             -
-              containerPort: 32293
+              containerPort: 9443
               protocol: "TCP"
           env:
             - name: HOST_NAME
@@ -4360,6 +4359,10 @@ function get_node_ip(){
       if [[ $(kubectl config current-context) = "minikube" ]]
       then
           NODE_IP=$(minikube ip)
+      elif [[ $(kubectl config current-context) = "docker-desktop" ]]
+      then
+          NODE_IP="127.0.0.1"
+      else
       else
         echo "We could not find your cluster node-ip."
         while [[ -z "$NODE_IP" ]]
@@ -4383,6 +4386,9 @@ function progress_bar(){
   pod_status=$(kubectl get pods -n wso2 -o jsonpath='{.items[?(@.metadata.labels.product=="api-manager")].status.conditions[*].status}')
 
   num_true_const=0; progress_unit="";num_true=0; time_proc=0;
+
+  # check for 250s until the pod is live.
+  time_limit=250 
 
   arr_dep=($dep_status); arr_pod=($pod_status)
 
@@ -4457,7 +4463,7 @@ function progress_bar(){
 
       sleep 1
 
-      if [[ $time_proc -gt 250 ]]
+      if [[ $time_proc -gt $time_limit ]]
       then
           echoBold "\n\nSomething went wrong! Please Follow \"https://wso2.com/products/install/faq/#Kubernetes\" for more information\n"
           exit 2
@@ -4534,9 +4540,9 @@ function deploy(){
     echoBold "Successfully deployed WSO2 API Manager.\n\n"
 
     echoBold "1. Try navigating to\n\n"
-    echoBold "\thttps://$NODE_IP:32293/carbon/\n"
-    echoBold "\thttps://$NODE_IP:32293/publisher/\n"
-    echoBold "\thttps://$NODE_IP:32293/devportal/\n"
+    echoBold "\thttps://$NODE_IP:30443/carbon/\n"
+    echoBold "\thttps://$NODE_IP:30443/publisher/\n"
+    echoBold "\thttps://$NODE_IP:30443/devportal/\n"
     echoBold "\thttps://$NODE_IP:30646/analytics-dashboard/\n\n"
     echoBold "from your favourite browser using credentials admin/admin\n\n"
 

--- a/simple/funcs
+++ b/simple/funcs
@@ -104,6 +104,9 @@ function get_node_ip(){
       if [[ $(kubectl config current-context) = "minikube" ]]
       then
           NODE_IP=$(minikube ip)
+      elif [[ $(kubectl config current-context) = "docker-desktop" ]]
+      then
+          NODE_IP="127.0.0.1"
       else
         echo "We could not find your cluster node-ip."
         while [[ -z "$NODE_IP" ]]
@@ -127,6 +130,9 @@ function progress_bar(){
   pod_status=$(kubectl get pods -n wso2 -o jsonpath='{.items[?(@.metadata.labels.product=="api-manager")].status.conditions[*].status}')
 
   num_true_const=0; progress_unit="";num_true=0; time_proc=0;
+
+  # check for 250s until the pod is live.
+  time_limit=250 
 
   arr_dep=($dep_status); arr_pod=($pod_status)
 
@@ -201,7 +207,7 @@ function progress_bar(){
 
       sleep 1
 
-      if [[ $time_proc -gt 250 ]]
+      if [[ $time_proc -gt $time_limit ]]
       then
           echoBold "\n\nSomething went wrong! Please Follow \"https://wso2.com/products/install/faq/#Kubernetes\" for more information\n"
           exit 2
@@ -278,9 +284,9 @@ function deploy(){
     echoBold "Successfully deployed WSO2 API Manager.\n\n"
 
     echoBold "1. Try navigating to\n\n"
-    echoBold "\thttps://$NODE_IP:32293/carbon/\n"
-    echoBold "\thttps://$NODE_IP:32293/publisher/\n"
-    echoBold "\thttps://$NODE_IP:32293/devportal/\n"
+    echoBold "\thttps://$NODE_IP:30443/carbon/\n"
+    echoBold "\thttps://$NODE_IP:30443/publisher/\n"
+    echoBold "\thttps://$NODE_IP:30443/devportal/\n"
     echoBold "\thttps://$NODE_IP:30646/analytics-dashboard/\n\n"
     echoBold "from your favourite browser using credentials admin/admin\n\n"
 

--- a/simple/funcs4opensource
+++ b/simple/funcs4opensource
@@ -84,6 +84,9 @@ function get_node_ip(){
       if [[ $(kubectl config current-context) = "minikube" ]]
       then
           NODE_IP=$(minikube ip)
+      elif [[ $(kubectl config current-context) = "docker-desktop" ]]
+      then
+          NODE_IP="127.0.0.1"
       else
         echo "We could not find your cluster node-ip."
         while [[ -z "$NODE_IP" ]]
@@ -107,6 +110,9 @@ function progress_bar(){
   pod_status=$(kubectl get pods -n wso2 -o jsonpath='{.items[?(@.metadata.labels.product=="api-manager")].status.conditions[*].status}')
 
   num_true_const=0; progress_unit="";num_true=0; time_proc=0;
+
+  # check for 250s until the pod is live.
+  time_limit=250 
 
   arr_dep=($dep_status); arr_pod=($pod_status)
 
@@ -236,10 +242,10 @@ function deploy(){
     echoBold "Successfully deployed WSO2 API Manager.\n\n"
 
     echoBold "1. Try navigating to\n\n"
-    echoBold "\thttps://$NODE_IP:32293/carbon/\n"
-    echoBold "\thttps://$NODE_IP:32293/publisher/\n"
-    echoBold "\thttps://$NODE_IP:32293/devportal/\n"
-    echoBold "\thttps://$NODE_IP:30643/analytics-dashboard/\n\n"
+    echoBold "\thttps://$NODE_IP:30443/carbon/\n"
+    echoBold "\thttps://$NODE_IP:30443/publisher/\n"
+    echoBold "\thttps://$NODE_IP:30443/devportal/\n"
+    echoBold "\thttps://$NODE_IP:30646/analytics-dashboard/\n\n"
     echoBold "from your favourite browser using credentials admin/admin\n\n"
 
     echoBold "2. Follow \"https://apim.docs.wso2.com/en/latest/GettingStarted/quick-start-guide/\" to start using WSO2 API Manager.\n\n"

--- a/simple/kubernetes-apim-analytics/dashboard/wso2am-pattern-1-analytics-dashboard-conf.yaml
+++ b/simple/kubernetes-apim-analytics/dashboard/wso2am-pattern-1-analytics-dashboard-conf.yaml
@@ -376,9 +376,9 @@ data:
         allScopes: apim_analytics:admin openid apim:api_view apim:subscribe apim_analytics:monitoring_dashboard:own apim_analytics:monitoring_dashboard:edit apim_analytics:monitoring_dashboard:view apim_analytics:business_analytics:own apim_analytics:business_analytics:edit apim_analytics:business_analytics:view apim_analytics:api_analytics:own apim_analytics:api_analytics:edit apim_analytics:api_analytics:view apim_analytics:application_analytics:own apim_analytics:application_analytics:edit apim_analytics:application_analytics:view
         adminUsername: admin
         adminPassword: admin
-        kmDcrUrl: https://"ip.node.k8s.&.wso2.apim":32293/client-registration/v0.17/register
-        kmTokenUrlForRedirection: https://"ip.node.k8s.&.wso2.apim":32293/oauth2
-        kmTokenUrl: https://"ip.node.k8s.&.wso2.apim":32293/oauth2
+        kmDcrUrl: https://wso2am-pattern-1-am-1-service:9443/client-registration/v0.17/register
+        kmTokenUrlForRedirection: https://"ip.node.k8s.&.wso2.apim":30443/oauth2
+        kmTokenUrl: https://wso2am-pattern-1-am-1-service:9443/oauth2
         kmUsername: admin
         kmPassword: admin
         portalAppContext: analytics-dashboard
@@ -386,9 +386,9 @@ data:
         cacheTimeout: 900
         baseUrl: https://"ip.node.k8s.&.wso2.apim":30646
         grantType: authorization_code
-        publisherUrl: https://"ip.node.k8s.&.wso2.apim":32293
-        devPortalUrl: https://"ip.node.k8s.&.wso2.apim":32293
-        externalLogoutUrl: https://"ip.node.k8s.&.wso2.apim":32293/oidc/logout
+        publisherUrl: https://wso2am-pattern-1-am-1-service:9443
+        devPortalUrl: https://wso2am-pattern-1-am-1-service:9443
+        externalLogoutUrl: https://"ip.node.k8s.&.wso2.apim":30443/oidc/logout
 
     wso2.dashboard:
       roles:

--- a/simple/kubernetes-apim-analytics/dashboard/wso2am-pattern-1-analytics-dashboard-deployment.yaml
+++ b/simple/kubernetes-apim-analytics/dashboard/wso2am-pattern-1-analytics-dashboard-deployment.yaml
@@ -61,11 +61,11 @@ spec:
                 command:  ['sh', '-c', '${WSO2_SERVER_HOME}/bin/dashboard.sh stop']
           resources:
             requests:
-              memory: 4Gi
-              cpu: 2000m
+              memory: 1Gi
+              cpu: 1000m
             limits:
-              memory: 4Gi
-              cpu: 2000m
+              memory: 1Gi
+              cpu: 1000m
           imagePullPolicy: Always
           securityContext:
             runAsUser: 802

--- a/simple/kubernetes-apim-analytics/worker/wso2apim-analytics-worker-deployment.yaml
+++ b/simple/kubernetes-apim-analytics/worker/wso2apim-analytics-worker-deployment.yaml
@@ -61,11 +61,11 @@ spec:
                 command:  ['sh', '-c', '${WSO2_SERVER_HOME}/bin/worker.sh stop']
           resources:
             requests:
-              memory: 4Gi
-              cpu: 2000m
+              memory: 1Gi
+              cpu: 1000m
             limits:
-              memory: 4Gi
-              cpu: 2000m
+              memory: 1Gi
+              cpu: 1000m
           imagePullPolicy: Always
           securityContext:
             runAsUser: 802

--- a/simple/kubernetes-apim/wso2apim-conf.yaml
+++ b/simple/kubernetes-apim/wso2apim-conf.yaml
@@ -10,7 +10,7 @@ data:
     [server]
     hostname = ""ip.node.k8s.&.wso2.apim""
     node_ip = "$env{NODE_IP}"
-    offset=22850
+    # offset=0
     mode = "single" #single or ha
     base_path = "${carbon.protocol}://${carbon.host}:${carbon.management.port}"
     #discard_empty_caches = false
@@ -155,7 +155,7 @@ data:
     #enable_token_hashing = false
 
     [apim.devportal]
-    url = "https://"ip.node.k8s.&.wso2.apim":32293/devportal"
+    url = "https://"ip.node.k8s.&.wso2.apim":30443/devportal"
     #enable_application_sharing = false
     #if application_sharing_type, application_sharing_impl both defined priority goes to application_sharing_impl
     #application_sharing_type = "default" #changed type, saml, default #todo: check the new config for rest api
@@ -269,4 +269,7 @@ data:
     username = "${admin.username}"
     password = "${admin.password}"
     'header.X-WSO2-KEY-MANAGER' = "default"
+
+    [transport.https.properties]
+    proxyPort = 30443
 ---

--- a/simple/kubernetes-apim/wso2apim-deployment.yaml
+++ b/simple/kubernetes-apim/wso2apim-deployment.yaml
@@ -51,7 +51,7 @@ spec:
               command:
                 - /bin/sh
                 - -c
-                - nc -z localhost 32293
+                - nc -z localhost 9443
             initialDelaySeconds: 250
             periodSeconds: 10
           readinessProbe:
@@ -59,7 +59,7 @@ spec:
               command:
                 - /bin/sh
                 - -c
-                - nc -z localhost 32293
+                - nc -z localhost 9443
             initialDelaySeconds: 250
             periodSeconds: 10
           lifecycle:
@@ -68,24 +68,24 @@ spec:
                 command:  ['sh', '-c', '${WSO2_SERVER_HOME}/bin/wso2server.sh stop']
           resources:
             requests:
-              memory: 2Gi
-              cpu: 2000m
+              memory: 1Gi
+              cpu: 1000m
             limits:
               memory: 2Gi
               cpu: 2000m
           imagePullPolicy: Always
           ports:
             -
-              containerPort: 31130
+              containerPort: 8280
               protocol: "TCP"
             -
-              containerPort: 31093
+              containerPort: 8243
               protocol: "TCP"
             -
-              containerPort: 32613
+              containerPort: 9763
               protocol: "TCP"
             -
-              containerPort: 32293
+              containerPort: 9443
               protocol: "TCP"
           env:
             - name: HOST_NAME

--- a/simple/kubernetes-apim/wso2apim-deployment.yaml
+++ b/simple/kubernetes-apim/wso2apim-deployment.yaml
@@ -71,8 +71,8 @@ spec:
               memory: 2Gi
               cpu: 2000m
             limits:
-              memory: 3Gi
-              cpu: 3000m
+              memory: 2Gi
+              cpu: 2000m
           imagePullPolicy: Always
           ports:
             -

--- a/simple/kubernetes-apim/wso2apim-service.yaml
+++ b/simple/kubernetes-apim/wso2apim-service.yaml
@@ -15,29 +15,25 @@ spec:
     -
       name: pass-through-http
       protocol: TCP
-      # for 8280 with a offset of 22850
-      port: 31130
-      targetPort: 31130
+      port: 8280
+      targetPort: 8280
       nodePort: 30280
     -
       name: pass-through-https
       protocol: TCP
-      # for 8243 with a offset of 22850
-      port: 31093
-      targetPort: 31093
+      port: 8243
+      targetPort: 8243
       nodePort: 30243
     -
       name: servlet-http
       protocol: TCP
-      # for 9763 with a offset of 22850
-      port: 32613
-      targetPort: 32613
+      port: 9763
+      targetPort: 9763
       nodePort: 30763
     -
       name: servlet-https
       protocol: TCP
-      # for 9443 with a offset of 22850
-      port: 32293
-      targetPort: 32293
-      nodePort: 32293
+      port: 9443
+      targetPort: 9443
+      nodePort: 30443
 ---

